app/Transformers/BibleFilePermissionsTransformer.php

```diff
@@ -21,7 +21,7 @@ public function transform($access)
          *      title="v4_bible_filesets_permissions.index",
          *      @OA\Xml(name="v4_bible_filesets_permissions.index"),
          *      @OA\Items(
-         *          @OA\Property(property="fileset_id",          ref="#/components/schemas/BibleFileset/properties/hash_id"),
```

Comment:
Instead of deleting the ref this can be `ref="#/components/schemas/BibleFileset/properties/id"`

Reply:
updated ref